core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -548,7 +538,24 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 	// it accepted the transaction despite the error and we can move forwards
 	// assuming success in this case.
 
-	eb.logger.Criticalw("Unknown error occurred while handling eth_tx queue in ProcessUnstartedEthTxs. This chain/RPC client may not be supported. Urgent resolution required, Chainlink is currently operating in a degraded state and may miss transactions", "err", sendError, "etx", etx, "attempt", attempt)
+	if sendError.IsTxFeeExceedsCap() {
+		lgr.Criticalw(fmt.Sprintf("Sending transaction failed; %s", label.RPCTxFeeCapConfiguredIncorrectlyWarning),
+			"etx", etx,
+			"attempt", attempt,
+			"err", sendError,
+			"id", "RPCTxFeeCapExceeded",
+		)
+		// Note that we may have broadcast to multiple nodes and had it
+		// accepted by one of them! It is not guaranteed that all nodes share
+		// the same tx fee cap. That is why we must treat this as an unknown
+		// error that may have been confirmed.
+		//
+		// If there is only one RPC node, or all RPC nodes have the same
+		// configured cap, this transaction will get stuck and keep repeating
+		// forever until the issue is resolved.
+	} else {
+		eb.logger.Criticalw("Unknown error occurred while handling eth_tx queue in ProcessUnstartedEthTxs. This chain/RPC client may not be supported. Urgent resolution required, Chainlink is currently operating in a degraded state and may miss transactions", "err", sendError, "etx", etx, "attempt", attempt)
```

Comment:
Would it make sense to use the `lggr` with context?
```suggestion
		lggr.Criticalw("Unknown error occurred while handling eth_tx queue in ProcessUnstartedEthTxs. This chain/RPC client may not be supported. Urgent resolution required, Chainlink is currently operating in a degraded state and may miss transactions", "err", sendError, "etx", etx, "attempt", attempt)
```

Reply:
yes!